data/mods/Magiclysm/mutations/mutations.json

```diff
@@ -8,5 +8,482 @@
+    "spells_learned": [ [ "dragon_breath_black", 0 ] ]
+  },
+  {
+    "type": "mutation",
+    "id": "THIRST",
+    "copy-from": "THIRST",
+    "extend": { "category": [ "DRAGON_BLACK" ] }
+  },
+  {
+    "type": "mutation",
+    "id": "THIRST2",
+    "copy-from": "THIRST2",
+    "extend": { "category": [ "DRAGON_BLACK" ] }
+  },
+  {
+    "type": "mutation",
+    "id": "CARNIVORE",
+    "copy-from": "CARNIVORE",
+    "extend": { "category": [ "DRAGON_BLACK" ] }
+  },
+  {
+    "type": "mutation",
+    "id": "COLDBLOOD3",
+    "copy-from": "COLDBLOOD3",
+    "extend": { "category": [ "DRAGON_BLACK" ] }
+  },
+  {
+    "type": "mutation",
+    "id": "COLDBLOOD4",
+    "copy-from": "COLDBLOOD4",
+    "extend": { "category": [ "DRAGON_BLACK" ], "threshreq": [ "THRESH_DRAGON_BLACK" ] }
+  },
+  {
+    "type": "mutation",
+    "id": "HISS",
+    "copy-from": "HISS",
+    "extend": { "category": [ "DRAGON_BLACK" ] }
+  },
+  {
+    "type": "mutation",
+    "id": "SNARL",
+    "copy-from": "SNARL",
+    "extend": { "category": [ "DRAGON_BLACK" ] }
+  },
+  {
+    "type": "mutation",
+    "id": "MANA_CRYST_MANA",
+    "copy-from": "MANA_CRYST_MANA",
+    "extend": { "category": [ "DRAGON_BLACK" ], "threshreq": [ "THRESH_DRAGON_BLACK" ] }
+  },
+  {
+    "type": "mutation",
+    "id": "DRAGON_MAGIC",
+    "name": { "str": "Draconic Magic" },
+    "description": "You are imbued with with the ancient magic powers of black dragons, improving your mana reserves and opening your mind to amazing feats of magic.",
+    "points": 3,
+    "category": [ "DRAGON_BLACK" ],
+    "threshreq": [ "THRESH_DRAGON_BLACK" ],
+    "purifiable": false,
+    "mana_modifier": 500
+  },
+  {
+    "type": "mutation",
+    "id": "DRAGON_SPELLS",
+    "name": { "str": "Draconic Spellcasting" },
+    "description": "You no longer need to move your arms or use your voice to cast spells, due to your draconic modifications.  Arm and mouth encumbrance no longer applies to spell failure chance.",
+    "points": 3,
+    "category": [ "DRAGON_BLACK" ],
+    "threshreq": [ "THRESH_DRAGON_BLACK" ],
+    "prereqs": [ "DRAGON_MAGIC" ],
+    "flags": [ "SUBTLE_SPELL", "SILENT_SPELL" ]
+  },
+  {
+    "type": "mutation",
+    "id": "MANA_SIPHON_3",
+    "copy-from": "MANA_SIPHON_3",
+    "extend": { "category": [ "DRAGON_BLACK" ], "threshreq": [ "THRESH_DRAGON_BLACK" ] }
+  },
+  {
+    "type": "mutation",
+    "id": "MANA_ADD_MANATOUCHED",
+    "copy-from": "MANA_ADD_MANATOUCHED",
+    "extend": { "category": [ "DRAGON_BLACK" ], "threshreq": [ "THRESH_DRAGON_BLACK" ] }
+  },
+  {
+    "type": "mutation",
+    "id": "MANA_REGEN_MANATOUCHED",
+    "copy-from": "MANA_REGEN_MANATOUCHED",
+    "extend": { "category": [ "DRAGON_BLACK" ], "threshreq": [ "THRESH_DRAGON_BLACK" ] }
+  },
+  {
+    "type": "mutation",
+    "id": "MANA_MULT_MANATOUCHED",
+    "copy-from": "MANA_MULT_MANATOUCHED",
+    "extend": { "category": [ "DRAGON_BLACK" ], "threshreq": [ "THRESH_DRAGON_BLACK" ] }
```

Comment:
making manatouched totally obsolete with this dragon mutation tree is a no-go.  adding some new acid-based spells i think would be a neater idea than grabbing all the mana pool mutations from manatouched. taking away the two extra special things manatouched had as two separate mutations (subtle and silent spell) also is way too much - you might be able to convince me you can have one, but i don't want to give both.

Reply:
it would definitely be subtle if anything, but Dragons don't have arms or standard vocalizations for spells and I was going under the concept that they would not be impeded by encumbrance specifically because they are becoming more innately magical as dragons are. I'll roll the pool, regen, and sensitivity to the next tier down and take out the silent spell as subtle spell synergizes better with the concept that dragons have adapted to using different or less intricate somatic components and will help negate the natural encumbrance innate to being large/huge and having scales.

as for spells, I was having a difficult time coming up with mutations to tie them to. Tying more than one spell to a single mutation seemed too strong, and tying one to each post-threshold mutation was difficult to try and think of thematically consistent spells to go with the mutation.

Reply:
just make the mutation only give you the spell, and call the mutation whatever the spell's name is. that's an easy way to give a mutation that gives a spell.

Reply:
The Dragon Spellcasting previously gave the rune spell, but I think either a buff or a debuff, maybe to mimic a Dragon's Terror Aura or something, or a power up spell or something that improves strength and gives some kind of regeneration? What are your thoughts?

Reply:
a debuff would be pretty thematic, and i won't say no to a buff spell